cli/command/export.go

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright (c) 2023 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2023-11-9
+ * Author: Jiang Jun (youarefree123)
+ */
+
+// __SIGN_BY_YOUAREFREE123__
+
+package command
+
+import (
+	"path"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/opencurve/curveadm/cli/cli"
+	comm "github.com/opencurve/curveadm/internal/common"
+	"github.com/opencurve/curveadm/internal/configure/topology"
+	"github.com/opencurve/curveadm/internal/errno"
+	"github.com/opencurve/curveadm/internal/playbook"
+	cliutil "github.com/opencurve/curveadm/internal/utils"
+	"github.com/spf13/cobra"
+)
+
+var (
+	GET_EXPORT_PLAYBOOK_STEPS = []int{
+		playbook.SYNC_CONFIG,
+		playbook.EXPORT_TOOLSV2_CONF,
+	}
+)
+
+type exportOptions struct {
+	output string
```

Comment:
which one host to export? Specify host?

Reply:
I found that all hosts include curve.yaml. I think the conf in each host should be the same, so currently I have selected dcs[0], just like the implementation in the “status” command.
[status.go](https://github.com/opencurve/curveadm/blob/580b26dd06144f327a9afe5e103bd18ff7a8a1a9/cli/command/status.go#L118-L120)